[10.0] Migrate mrp_bom_note

bimajatiwijaya opened the pull request.

bimajatiwijaya committed:
[10.0] Migrate mrp_bom_note

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -1,22 +1,41 @@
+
+To install this module, you need to:
+
+#. install mrp application
```
Here: not really need for enumerated list (there is only one element) ...

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -1,22 +1,41 @@
 =====
 
-* Go to Manufacturing > Products > Bill of Materials
+* Go to Manufacturing > Master Data > Bill of Materials
```
... but here it would be convenient actually to have one (enumerated list)

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -1,22 +1,41 @@
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/129/9.0
 
+Known issues / Roadmap
```
if empty please delete the section

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -28,12 +47,18 @@ If you spotted it first, help us smashing it by providing a detailed and welcome
 * Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 * Ana Juaristi <anajuaristi@avanzosc.es>
+* Bima Wijaya <bimajatiwijaya@gmail.com>
```
below please update http=>https

elicoidal reviewed (changes requested):
small details. LGTM

bimajatiwijaya committed:
Revise README.srt :
1. Delete empty section.
2. Add enumerated on usage section.
3. Removed enumerated on installation section.
4. Change to https://odoo-community.org

bimajatiwijaya commented:
Thanks for your review @elicoidal . i update it.

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -1,17 +1,28 @@
+Configuration
+=============
+
+No need configure required.
```
No configuration required

elicoidal commented on mrp_bom_note/README.rst:
```diff
@@ -1,17 +1,28 @@
 
-* Go to Manufacturing > Products > Bill of Materials
-* Edit an existing BoM and fill in the Notes field on tab Notes
+1. Go to Manufacturing > Master Data > Bill of Materials then edit an existing BoM and fill in the Notes field on tab Notes
```
you misunderstood my point:
```
#. Go to Manufacturing > Master Data > Bill of Materials then 
#. Edit an existing BoM and 
#. Fill in the Notes field on tab Notes

bimajatiwijaya committed:
Fix changes requested : wrong sentence and usage section.

andhit-r commented on mrp_bom_note/__manifest__.py:
```diff
@@ -14,6 +14,7 @@
         "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
         "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
         "Ana Juaristi <anajuaristi@avanzosc.es>",
+        "Bima Wijaya <bimajatiwijaya@gmail.com>",
```
You should remove contributor key. Contributor list already mention on README

bimajatiwijaya committed:
removed contributor key from manifest. Check contributor on README.rst

andhit-r reviewed (approved):
LGTM

LoisRForgeFlow commented:
@elicoidal Could you update your review?

elicoidal reviewed (approved):
LGTM

LoisRForgeFlow commented:
commits squashed on merge
